Wiki-Contribute-Fork.md

```diff
@@ -0,0 +1,81 @@
+# How to get a fork of our wiki and keep it syncronized
+
+These steps are standard for contributing not only to our wiki but any other project once you understand what is going on.
+
+## Steps to follow
+
+### 1\. **Fork it** :fork_and_knife:
```

Comment:
I think they were there before. But to answer that it would be so is parted as is instead of convert it to a list, that way it would stay as a heading.